app/src/component/fileDownloadButton/view.js

```diff
@@ -76,7 +75,10 @@ class FileDownloadButton extends React.PureComponent {
           if (NativeModules.Firebase) {
             NativeModules.Firebase.track('purchase_uri', { uri: uri });
           }
-          purchaseUri(uri, onStartDownloadFailed);
+          purchaseUri(uri, costInfo, !isPlayable);
+          if (NativeModules.UtilityModule) {
```

Comment:
1) This check appears enough times that it may be worth wrapping.
2) What happens if this isn't true / when would it not be true?

Reply:
1. True.
2. Technically, it should never happen, because it's my responsibility to ensure UtilityModule exists in native code.